examples/bpk-component-swap-button/examples.js

```diff
@@ -27,4 +27,19 @@ const DefaultExample = () => (
   </div>
 );
 
-export default DefaultExample ;
+const CanvasDefaultExample = () => (
+
+  <BpkDarkExampleWrapper padded style={{padding:'2rem'}} >
+  <BpkSwapButton swapButtonStyle='canvas-default' ariaLiveTextProp='Swapped' onClick={action('Button clicked')}/>
+  </BpkDarkExampleWrapper>
+);
+const CanvasContrastExample = () => (
+  <div>
+  <BpkDarkExampleWrapper padded style={{padding:'2rem'}}>
+    <BpkSwapButton swapButtonStyle='canvas-contrast' ariaLiveTextProp='Swapped' onClick={action('Button clicked')}/>
```

Comment:
Here I would expect to see for `swapButtonStyle` to be `SWAPBUTTON_STYLES.canvasContrast`